src/badge/badge.scss

```diff
@@ -11,14 +11,17 @@ $iui-badge-background-primary: $iui-color-foreground-primary--dark;
 $iui-badge-text-color: rgba($iui-color-foreground-body--light, $iui-opacity-3--light);
 
 @mixin iui-badge {
+  --_iui-badge-background-color: #{$iui-badge-background-default};
+  --_iui-badge-text-color: #{$iui-badge-text-color};
```

Comment:
Have you updated "--badge-color" in Badge.tsx and Badge.test.tsx in the React project?

Reply:
That happens after the CSS PR is merged and released